charts.d/squid.chart.sh

```diff
@@ -113,8 +114,8 @@ squid_update() {
 	# even if something goes wrong, no other code can be executed
 
 	eval "$(squid_get_stats |\
-		 sed -e "s/ \+/ /g" -e "s/\./_/g" -e "s/^\([a-z0-9_]\+\) *= *\([0-9]\+\)$/local squid_\1=\2/g" |\
-		egrep "^local squid_(client_http|server_all)_[a-z0-9_]+=[0-9]+$")"
+		 sed -e "s/ \\+/ /g" -e "s/\\./_/g" -e "s/^\\([a-z0-9_]\\+\\) *= *\\([0-9]\\+\\)$/local squid_\\1=\\2/g" |\
```

Comment:
why?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC1117

Reply:
Due to https://github.com/koalaman/shellcheck/wiki/SC1117 which should be retired, but isn't in latest shellcheck version

Reply:
I don't think it will work. Have your tried it?

Reply:
No. I will revert that.